packages/transformer/src/IModelTransformer.ts

```diff
@@ -2375,6 +2395,17 @@ export class IModelTransformer extends IModelExportHandler {
     this.finalizeTransformation();
   }
 
+  /** Changeset data must be initialized in order to build correct changeOptions. Call [[IModelTransformer.initialize]] for initialization of synchronization provenance data */
+  private getExportChangesOptions(accessToken?: AccessToken, startChangesetId?: string): ExportChangesOptions {
```

Comment:
nit: worth thinking about if we can assert something to be sure this is called after change data is processed... like add a new private variable that is set to true after changedata is processed... or make this._changesetRanges have a third empty state